plugins/modules/cloudflare_dns.py

```diff
@@ -625,6 +650,9 @@ def delete_dns_records(self, **kwargs):
                     if not self.module.check_mode:
                         result, info = self._cf_api_call('/zones/{0}/dns_records/{1}'.format(rr['zone_id'], rr['id']), 'DELETE')
             else:
+                if ((rr['type'] == 'CAA') and (rr['data']['tag'] != params['caa_tag'])):
```

Comment:
These brackets are superfluous in Python and make it harder to read the expression IMO:
```suggestion
                if rr['type'] == 'CAA' and rr['data']['tag'] != params['caa_tag']:
```